docs/tutorial/part03.rst

```diff
@@ -2,5 +2,295 @@
 How to create the base models.
 ==============================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Create a new app for the API: 
+==============================
+Right now, we need to create  a Django app which is going to be use to create the search API and,

```

Comment:
Extra space before the `a` and a question; do you think that we need `to be used`?
Sorry just to know 😅